README.md

```diff
@@ -139,24 +139,26 @@ bot.on('message', async message => {
     await bot.wait(1000);
     
     // ---- send buttons (single card)
+    //make sure you import Buttons from facebook-messenger-bot
     buttons = new Buttons();
     buttons.add({text: 'Google', url: 'http://google.com'});
     buttons.add({text: 'Yahoo', url: 'http://yahoo.com'});
     buttons.add({text: 'Bing', url: 'http://bing.com'});
     out = new Elements();
     out.add({text: 'search engines', subtext: 'click to get redirected', buttons}); // add a card
-    await bot.send(to, out);
+    await bot.send(sender.id, out);
     
     await bot.wait(2000);
     
     // ---- send image + buttons (multiple cards)
+    //make sure you import Buttons from facebook-messenger-bot
```

Comment:
Rather than a comment, can you please make it at the import level?

Change from:
```js
import {Elements} from 'facebook-messenger-bot'; // import Bot class
```

to 
```js
import {Elements, Buttons, ...whatever else you need} from 'facebook-messenger-bot'; // import Bot class
```